Healthy/Classes/Utilities/Logger/FileSystemLogger.swift

```diff
@@ -0,0 +1,38 @@
+import Foundation
+final class FileSystemLogger: Logging {
```

Comment:
```suggestion
import Foundation

final class FileSystemLogger: Logging {
```

Please keep a message after import statement and between methods!